src/Mobs/Monster.cpp

```diff
@@ -397,32 +347,29 @@ void cMonster::SetPitchAndYawFromDestination()
 	{
 		if (m_Target->IsPlayer())
 		{
-			FinalDestination.y = ((cPlayer *)m_Target)->GetStance();
+			FinalDestination.y = static_cast<cPlayer *>(m_Target)->GetStance() - 1;
```

Comment:
Your PR is pretty good, I could take it from here and take care of those little things myself if you merge it. If it's currently pathing, you can merge it.